stages/org.osbuild.ostree.aleph

```diff
@@ -52,7 +52,7 @@ def aleph_container(tree, imgref):
     container_data_config = json.loads(container_data_config_json)
 
     aleph_digest = container_data['config']['digest']
-    aleph_ref = f"docker://{imgref}"
+    aleph_ref = imgref
```

Comment:
This is a behaviour change which is not reflected by an input option change, which we generally don't allow.  Given how specific this stage is, and how it's probably not used anywhere outside of FCOS, I think it's okay.